draft-ietf-tls-key-share-prediction.md

```diff
@@ -30,24 +30,25 @@ author:
 normative:
 
 informative:
+...
 
 --- abstract
 
-This document defines a mechanism for servers to communicate key share preferences in DNS. Clients may use this information to reduce TLS handshake round-trips.
+This document defines a mechanism for servers to communicate supported key share algorithms in DNS. Clients may use this information to reduce TLS handshake round-trips.
 
 --- middle
 
 # Introduction
 
-Named groups in TLS 1.3 {{!RFC8446}} are negotiated with two lists in the ClientHello: The client sends its full preferences in the `supported_groups` extension, but also generates key shares for a subset in the `key_share` extension. Named groups in this subset may be used in one, while named groups outside the subset requires a HelloRetryRequest and two round trips. The additional round trip is undesirable for performance, but unused key shares consume network and computational resources, so clients often do not generate key shares for all groups.
+Named groups in TLS 1.3 {{!RFC8446}} are negotiated with two lists in the ClientHello: The client sends its full preferences in the `supported_groups` extension, but also generates key shares for a subset in the `key_share` extension. Named groups in this subset can be used in one round trip, while named groups outside the subset require a HelloRetryRequest and hence two round trips. The additional round trip is undesirable for performance, but unused key shares consume network and computational resources, so clients often do not generate key shares for all groups.
```

Comment:
Sorry for a passing-by comment, if you prefer I share the comments in the TLS mailing list.

In the previous version we were talking about "key share preferences in DNS". Then, from "The client sends its full preferences" it was clear that you talk about the same preferences. Now you've replaced "preferences" with "supported {..} algorithms". Would it make sense to update it here as well?